conf/layer.conf

```diff
@@ -8,7 +8,22 @@ BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
 BBFILE_COLLECTIONS += "celebrity-face-match"
 BBFILE_PATTERN_celebrity-face-match = "^${LAYERDIR}/"
 BBFILE_PRIORITY_celebrity-face-match = "24"
-LAYERSERIES_COMPAT_celebrity-face-match = "hardknott"
-LAYERDEPENDS_celebrity-face-match = "phytec ampliphy meta-ml"
+LAYERSERIES_COMPAT_celebrity-face-match = "scarthgap"
+LAYERDEPENDS_celebrity-face-match = "phytec ampliphy imx-machine-learning"
 
-DISTRO_VERSION_append = "-CelebrityFaceMatch-v0.6"
+BBMASK += "meta-ampliphy/recipes-graphics/wayland/weston-init.bbappend"
+BBMASK += "meta-imx/meta-bsp/recipes-graphics/wayland/weston-init.bbappend"
+BBMASK += "meta-freescale/recipes-graphics/wayland/weston-init.bbappend"
+BBMASK += "meta-freescale-distro/recipes-graphics/wayland/weston-init.bbappend"
+
+DISTRO_VERSION:append = "-CelebrityFaceMatch-v0.7"
```

Comment:
#5 uses the v0.7 already. For quite some other changes, this seems not to be based on the kirkstone WIP?!